src/checkoutservice/main.go

```diff
@@ -412,6 +412,18 @@ func (cs *checkoutService) prepOrderItems(ctx context.Context, items []*pb.CartI
 	cl := pb.NewProductCatalogServiceClient(conn)
 
 	for i, item := range items {
+		if item.GetProductId() == "HQTGWGPNH4" {
+			err := fmt.Errorf("invalid product #%q", item.GetProductId())
+
+			hub := sentry.CurrentHub()
+			hub.Client().CaptureException(
```

Comment:
sentry-go currently doesn't have a good support for capturing errors (manually or automatically) in the OTel context. This issue is partially tracked in https://github.com/getsentry/sentry-go/issues/555